src/main/java/xyz/nucleoid/dungeons/dungeons/assets/DgModelGenerator.java

```diff
@@ -0,0 +1,89 @@
+package xyz.nucleoid.dungeons.dungeons.assets;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import net.fabricmc.loader.api.FabricLoader;
+import net.minecraft.item.Item;
+import net.minecraft.util.registry.Registry;
+import xyz.nucleoid.dungeons.dungeons.util.item.DgItemModelRegistry;
+import xyz.nucleoid.dungeons.dungeons.util.item.DgItemUtil;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.LinkedList;
+import java.util.Map;
+
+// Todo: make better
+public class DgModelGenerator {
+    private static final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
```

Comment:
Shouldn't this be in Dungeons modinit as it might be used later on by other classes?